README.rst

```diff
@@ -25,12 +25,13 @@ Dependencies
 
 The required dependencies to use the software are:
 
-* Python >= 2.6,
+* Python >= 2.6
 * setuptools
 * Numpy >= 1.6.1
 * SciPy >= 0.9
 * Nibabel >= 1.1.0
 * Nilearn >= 0.1.3
+* Pandas >= 0.14
```

Comment:
Depending on which version of pandas ends up being required this line should be eventually changed accordingly.